com.unity.toonshader/Documentation~/Outline.md

```diff
@@ -39,43 +38,47 @@ Specifies how the inverted-outline objects spawn.
 ### Outline Width
 Specifies the width of the outline. This value relies on the scale when importing the model to Unity
 
-<img src="images/OutlineWidth.gif" height="256">
-
 ### Outline Color
 Specifies the color of outline.
 
-<img src="images/OutlineColor.gif" height="256">
-
-
 ### Blend Base Color to Outline
 Blends **Base Color** into outline color. 
 
-
-| Blend Base Color to Outline Off | Blend Base Color to Outline  On |
-| -- | -- | 
-| <img src="images/BlendBaseColorToOutlineOff.png" height="200"> | <img src="images/BlendBaseColorToOutlineOn.png" height="200"> |
+<canvas class="image-comparison" role="img" aria-label="A chibi-style character model with a thick outline. With Blend Base Color to Outline disabled, the outline is black. With Blend Base Color to Outline enabled, the outline matches the colors of the adjacent texture.">
+    <img src="images/BlendBaseColorToOutlineOff.png" title="Blend Base Color to Outline disabled">
+    <img src="images/BlendBaseColorToOutlineOn.png" title="Blend Base Color to Outline enabled">
+</canvas>
+<br />
+Drag the slider to compare the images.
 
 ### Outline Width Map
 Outline Width Map as gray scale Texture : Texture(linear). UTS provides  meticulous control solution for outline thickness as a texture map. Look at the difference of the outlines around the character's eyes and face.
 
-| Outline Width Map Off | Outline Width Map On |
-| -- | -- | 
-| <img src="images/OutlineWidthMapOff.png" width="320"> | <img src="images/OutlineWidthMapOn.png" width="320"> |
+<canvas class="image-comparison" role="img" aria-label="A chibi-style character model with rabbit ears. The model has outlining which becomes thinner, and the outlines around the eyes of the character disappear.">
+    <img src="images/OutlineWidthMapOff.png" title="Outline Width Map disabled">
+    <img src="images/OutlineWidthMapOn.png" title="Outline Width Map enabled">
+</canvas>
+<br />
+Drag the slider to compare the images.
 
 You will notice that the outlines of the white areas appear thicker in the texture map.
 
-| Outline Width Map applied above | Base Map |
-| -- | -- |
-| <img src="images/utc_all2_outlinesmpler.png" height="256"> | <img src="images/utc_all2_light.png" height="256"> |
 
+![The UV map texture that contains all the parts of the model.](images/utc_all2_light.png)<br/>
+An example of a Base Map.
 
+![A square grayscale texture that matches the Base Map. Most of the map is a consistent light gray. The face, legs, and hands are a darker gray. The white areas of the Base Map in the top-left are now black.](images/utc_all2_outlinesmpler.png)<br/>
+The Outline Width Map applied above.
```

Comment:
I think it's preferable for these two images to be displayed at the same size as before.

Reply:
No worries - they're resized and updated now.

Reply:
Ok, we'll fix this after this PR

Reply:
Ok - but it's very important not to reintroduce accessibility issues. You'll need to combine the images into a side-by-side image, or find another way that remains accessible.